wscript

```diff
@@ -902,7 +902,15 @@ standalone_features = [
         'desc': 'Apple Remote support',
         'deps': [ 'cocoa' ],
         'func': check_true
-    }
+    }, {
+        'name': '--macos-touchbar',
+        'desc': 'macOS Touch Bar support',
+        'deps': [ 'cocoa' ],
+        'func': check_cc(
+            fragment=load_fragment('touchbar.m'),
+            compile_filename='test-touchbar.m',
+            linkflags='-fobjc-arc')
+     }
```

Comment:
i just tried this and it didn't work. maybe because one is not supposed to included "AppKit/NSTouchBar.h" directly but rather use the whole framework "AppKit/AppKit.h". i wasn't too sure about this, maybe it's better to alloc and init a NSTouchbar and see if that fails as a check?